src/usr/local/www/pkg_mgr.php

```diff
@@ -130,12 +128,12 @@
 //		display_top_tabs($tab_array);
 }
 
-if (!$pkg_info || !is_array($pkg_info)):
-?>
+if (!$pkg_info || !is_array($pkg_info)):?>
```

Comment:
I have yet to find a text editor that understands PHP "alternate" control format :)